### PR TITLE
Added the homepage

### PR DESCRIPTION
**Pull request by aka001**
@noplay Kindly review the pr.

### CODE REVIEW

File: gns3server/handlers/__init__.py [hunk 1]
```diff
@@ -28,6 +28,20 @@
 from gns3server.handlers.api.config_handler import ConfigHandler
 from gns3server.handlers.api.server_handler import ServerHandler
 from gns3server.handlers.upload_handler import UploadHandler
+from ..config import Config
```
Comment (noplay) [inline gns3server/handlers/__init__.py]: I think it's not require

**Comment by noplay** [comment]
Thanks!

We can add pointers for usefull ressources:
* GN3 homepage http://gns3.com
* Support & Documentation: https://community.gns3.com/
* API documentation: http://api.gns3.net

**Comment by noplay** [comment]
It's related to #99

**Comment by aka001** [comment]
@noplay  I have updated the pr and squashed the commits.
Do you want me to add the links for the "/upload" page on "home page", so that user can move there without anybody's help?

**Comment by noplay** [comment]
Good idea for the /upload

**Comment by aka001** [comment]
@noplay I have updated the pr. Kindly review it.

File: gns3server/templates/homepage.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+<title>GNS3 Server</title>
+</head>
+<body>
+{% block body %}{% endblock %}
+</body>
+<h3>
+    Welcome to Home page of GNS3.
+</h3>
+<h4>
+    Visit the upload page to try GNS3 features: <a href = "http://127.0.0.1:8000/upload">Upload</a>
```
Comment (noplay) [inline gns3server/templates/homepage.html]: You could put just /upload because the user can change the host and port.

Comment (aka001) [inline gns3server/templates/homepage.html]: @noplay True that. I have updated the pr and squashed the commits. Kindly review it.

**Comment by aka001** [comment]
@noplay Is there any problem with the pr? Shall I update it ?

**Comment by noplay** [comment]
Sorry with the merge the PR has been autoclosed

**Comment by noplay** [comment]
Can you make a new PR against master?

Thanks and sorry for the auto close

**Comment by aka001** [comment]
@noplay I am having my exams today. I will create the pr by tonight. Sorry for the delay.

**Comment by aka001** [comment]
@noplay I have created a new pull request. Kindly review it.
https://github.com/GNS3/gns3-server/pull/106

**Pull request closed without merging**